tests/contenttypes_tests/test_management.py

```diff
@@ -71,3 +69,10 @@ def test_contenttypes_removed_in_apps_without_models(self):
             call_command('remove_stale_contenttypes', verbosity=2)
         self.assertIn("Deleting stale content type 'no_models | Fake'", stdout.getvalue())
         self.assertEqual(ContentType.objects.count(), self.before_count)
+
+    def test_contenttypes_removed_for_apps_not_in_installed_apps(self):
+        ContentType.objects.create(app_label='stale_app', model='Fake')
+        with mock.patch('builtins.input', return_value='yes'), captured_stdout() as stdout:
+            call_command('remove_stale_contenttypes', include_stale_apps=True, verbosity=2)
+        self.assertIn("Deleting stale content type 'stale_app | Fake'", stdout.getvalue())
+        self.assertEqual(ContentType.objects.count(), self.before_count)
```

Comment:
We should check if it works together with deleted models for installed apps. I would add `ContentType.objects.create(app_label='no_models', model='Fake 1')` to this test, e.g.
```python
@modify_settings(INSTALLED_APPS={'remove': ['empty_models']})
def test_contenttypes_removed_for_apps_not_in_installed_apps(self):
    ContentType.objects.create(app_label='no_models', model='Fake 1')
    ContentType.objects.create(app_label='empty_models', model='Fake 2')
    with mock.patch('builtins.input', return_value='yes'), captured_stdout() as stdout:
        call_command('remove_stale_contenttypes', include_stale_apps=True, verbosity=2)
    self.assertIn("Deleting stale content type 'no_models | Fake 1'", stdout.getvalue())
    self.assertIn("Deleting stale content type 'empty_models | Fake 2'", stdout.getvalue())
    self.assertEqual(ContentType.objects.count(), self.before_count)
```